cpp/src/io/parquet/reader_impl_helpers.hpp

```diff
@@ -202,9 +217,12 @@ class aggregate_reader_metadata {
 
   [[nodiscard]] auto get_num_row_groups() const { return num_row_groups; }
 
-  [[nodiscard]] auto const& get_schema(int schema_idx) const
+  [[nodiscard]] auto const& get_schema(int schema_idx, int pfm_idx = 0) const
   {
-    return per_file_metadata[0].schema[schema_idx];
+    CUDF_EXPECTS(
```

Comment:
We will never hit this error with the code in this PR. We can still keep to check against an invalid `schema_idx` of `pfm_idx` from elsewhere else I can remove it!